README.md

```diff
@@ -35,8 +45,8 @@ Run `cm/gen-config-files.sh`; ie. `./cm/gen-config-files.sh views/default.json`.
 
 ### Compiles and hot-reloads for development
```

Comment:
This section doesn't seem to include the instructions to compile the `auth-server` ts before spinning up the docker image?

Reply:
In my opinion, the docker build should take care of this, if not, I'd consider it a bug.
Hopefully, https://github.com/dappley/dappley-stats/pull/15/commits/6cfa1634b0aba09012004690ff295d0cce0b0345, resolved your issue and if not, we should revisit that.

Reply:
I think docker build only copy the files to the image.
https://github.com/dappley/dappley-stats/blob/user-authentication/cm/templates/docker/auth-server/Dockerfile.mustache#L5

Would adding `tsc` command in `start` script in https://github.com/dappley/dappley-stats/commit/6cfa1634b0aba09012004690ff295d0cce0b0345 cause any issue? `typescript` is a `devDependency`. Should it be a `dependency`?

Reply:
Seems like you need to explicitly say you don't want the `devDependencies` for them to not be installed,
https://yarnpkg.com/lang/en/docs/cli/install/#toc-yarn-install-production-true-false.
Which begs the question, should I be installing it using the production flag and compiling beforehand? I don't think it matter too much unless you want to speed up the build.

Reply:
d35e04e